### ISSUE TITLE
settings: data folder location

### ISSUE THREAD

**Issue by mendomusic** [hydrogen-music/hydrogen#448]
I would like to suggest to give the users options to relocate the local data folder.

Example:

from
`$HOME/.hydrogen/data`

to
`/external_drive/hydrogen/data`

Reference:
version 0.9.6.1

**Comment by xjjx** [comment]
Most users don't need this, but if you really need this, you can use symlinks.

**Comment by thijz** [comment]
actually dis is a duplicate of #243
and i personally find it quite relevant
also note that H2 is not only used on linux so the use of 'symlinks' is probably confusing for a lot of users

**Comment by xjjx** [comment]
symlinks are supported on M$shit and OSX also ( according to wikipedia - I didn't test myself ) and of course on all POSIX systems. Personally, I would like to see more commits on stability/improve code/resolving bugs, than fancy new features.

**Comment by thijz** [comment]
point taken, but let me rephrase my comment : 
- my brother who runs H2 on a mac has absolutely no idea what a symlink is, ket alone how to create one 
- same goes for a friend of mine who runs H2 on windows

IMHO this also isnt a fancy feature but a very basic one, it's just a pain when you have to browse to the correct folder each time you want to save a song and it's one of the first things that my brother started complaining about

**Comment by xjjx** [comment]
I don't belive that this feature will bring new users and (better) new developers to H2. After all this is for more advanced users... but I'm also insensitive for problems of users of closed-platforms ;-)
After all this is open source - so if someone will really need this , he write the code and give a pull request ... or he donate some developer to made this ;-)
